library/bignum_mod_raw.h

```diff
@@ -169,7 +169,6 @@ void mbedtls_mpi_mod_raw_sub( mbedtls_mpi_uint *X,
                               const mbedtls_mpi_uint *A,
                               const mbedtls_mpi_uint *B,
                               const mbedtls_mpi_mod_modulus *N );
-
```

Comment:
Hmm, this change here seems to cause this PR to conflict with #6742, so I think it would be best to undo it